mule-migration-tool-e2e-tests/src/test/resources/e2e/http/httpStatusCode/output_nc/src/main/mule/mule-config.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:http="http://www.mulesoft.org/schema/mule/http" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
+
+    <http:listener-config name="HTTP_Listener_Configuration" doc:name="HTTP Listener Configuration">
+        <http:listener-connection host="0.0.0.0" port="8081" />
+    </http:listener-config>
+
+    <http:request-config name="requesterConfig">
+        <http:request-connection protocol="HTTPS" host="httpbin.org" port="443" />
+    </http:request-config>
+
+    <flow name="mainFlow">
+        <http:listener config-ref="HTTP_Listener_Configuration" path="/api" doc:name="HTTP">
+            <http:response statusCode="#[message.attributes.statusCode]">

```

Comment:
should be `vars.outbound_http.status`

Reply:
done in new rev, it was a bug related on how we are storing the context on the copyProperties, now we store the value as the name of the set-variable variable (outbound_key_name) instead of the inbound translation